docs/ref/csrf.txt

```diff
@@ -150,12 +150,13 @@ Finally, you'll need to set the header on your AJAX request. Using the
 
     const request = new Request(
         /* URL */,
-        {headers: {'X-CSRFToken': csrftoken}}
+        {
+            headers: {'X-CSRFToken': csrftoken},
+            method: 'POST',
```

Comment:
I'd swap them
```suggestion
            method: 'POST',
            headers: {'X-CSRFToken': csrftoken},
```

Reply:
The suggested change makes the fields match the order they appear in the documentation for the [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) object. I agree.